content/src/main/content/jcr_root/apps/acs-commons/touchui-widgets/showhidedialogfields/source/showhidedialogfieldstabs.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * Extension to the standard dropdown/select and checkbox component. It enables hidding/unhidding of multiple dialog fields 
+ * and dialog tabs based on the selection made in the dropdown/select or on checkbox check or their combination. 
+ *
+ * How to use:
+ * - add the empty property cq-dialog-dropdown-checkbox-showhide to the dropdown/select or checkbox element

```

Comment:
this property name (and the data attribute) should probably be prefixed

Reply:
added acs- as prefix to property/data attributes names